packages/ember-data/lib/system/snapshot.js

```diff
@@ -12,7 +12,7 @@ var get = Ember.get;
   @constructor
   @param {DS.Model} internalModel The model to create a snapshot from
 */
-export default function Snapshot(internalModel) {
```

Comment:
I'd personally prefer to remove the one at the end, but not a big deal either way....